### PR TITLE
[ConstitutiveLawsApp] Correcting minor things and cleaning of kin plasticity integrator

### PR DESCRIPTION
**Pull request by AlejandroCornejo**
That's it. I does not break any tests since they are optimizatioin changes

### CODE REVIEW

File: applications/ConstitutiveLawsApplication/custom_constitutive/constitutive_laws_integrators/generic_constitutive_law_integrator_kinematic_plasticity.h [hunk 1]
```diff
@@ -221,7 +222,7 @@ class GenericConstitutiveLawIntegratorKinematicPlasticity
         }
 
         if (iteration > max_iter) {
-            KRATOS_WARNING_FIRST_N("Backward Euler Plasticity", 20) << "Maximum number of iterations in plasticity loop reached..." << std::endl;
+            KRATOS_WARNING("Backward Euler Plasticity", 20) << "Maximum number of iterations in plasticity loop reached..." << std::endl;
```
Comment (loumalouomega) [inline applications/ConstitutiveLawsApplication/custom_constitutive/constitutive_laws_integrators/generic_constitutive_law_integrator_kinematic_plasticity.h]: That's a lot of warnings (for each GP)

**Review by loumalouomega — DISMISSED** [review]
OK

File: applications/ConstitutiveLawsApplication/custom_constitutive/constitutive_laws_integrators/generic_constitutive_law_integrator_kinematic_plasticity.h (hunk 1 above)
Comment (AlejandroCornejo) [reply applications/ConstitutiveLawsApplication/custom_constitutive/constitutive_laws_integrators/generic_constitutive_law_integrator_kinematic_plasticity.h]: Then problem Vicente is that we have noticed that with the former warning they do not appear... In any case when this happens the simulation is prone to fail... I could put even a KratosError...

Comment (loumalouomega) [reply applications/ConstitutiveLawsApplication/custom_constitutive/constitutive_laws_integrators/generic_constitutive_law_integrator_kinematic_plasticity.h]: Yes, this type of warning should be improved to be able to restart it

**Review by SergioJimenezReyes — APPROVED** [review]
I have tested previous cases and all seems to be fine!

**Pull request merged**